nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -3766,7 +3765,7 @@ public void updateFlow(final VersionedFlowSnapshot proposedSnapshot, final Strin
                            final boolean updateDescendantVersionedFlows) {
         writeLock.lock();
         try {
-            verifyCanUpdate(proposedSnapshot, true, verifyNotDirty);
+             verifyCanUpdate(proposedSnapshot, true, verifyNotDirty);
```

Comment:
It looks like the changes in this file are just formatting related, should the changes be reverted?

Reply:
Strange - I don't think I touched this file. I'll revert it.